assets/config/config.yaml

```diff
@@ -1,18 +1,18 @@
 operator:
-  scyllaDBVersion: "6.2.0"
+  scyllaDBVersion: "docker.io/scylladb/scylla:2025.1.2@sha256:84e914792c61a7703ff616bf6ee6d4becbcf68845221bd2458fcab10ef64302c"
   # scyllaDBEnterpriseVersionNeedingConsistentClusterManagementOverride sets enterprise version
   # that requires consistent_cluster_management workaround for restore.
   # In the future, enterprise versions should be run as a different config instance in its own run.
   scyllaDBEnterpriseVersionNeedingConsistentClusterManagementOverride: "2024.1.11"
-  scyllaDBUtilsImage: "docker.io/scylladb/scylla:6.2.0@sha256:5b53a7c60d9f9555bb87791ff29b2e633c6f472aec00de7afaf4db1addc6d594"
-  scyllaDBManagerVersion: "3.4.0@sha256:6424c36929b39ef4e679ee7d61578b1070e28c186b23b278147fde8e391d9db1"
-  scyllaDBManagerAgentVersion: "3.4.0@sha256:441403aed8880cad1feef68aa7a8ee9ffd99a458dc1dcff3dc54ce5bf3cb07b7"
-  bashToolsImage: "registry.access.redhat.com/ubi9/ubi:9.5-1731517889@sha256:2bae9062eddbbc18e76555972e7026ffe02cef560a0076e6d7f72bed2c05723f"
-  grafanaImage: "docker.io/grafana/grafana:11.3.0@sha256:a0f881232a6fb71a0554a47d0fe2203b6888fe77f4cefb7ea62bed7eb54e13c3"
-  grafanaDefaultPlatformDashboard: "scylladb-6.2/scylla-overview.6.2.json"
-  prometheusVersion: "v2.54.1"
+  scyllaDBUtilsImage: "docker.io/scylladb/scylla:2025.1.2@sha256:84e914792c61a7703ff616bf6ee6d4becbcf68845221bd2458fcab10ef64302c"
+  scyllaDBManagerVersion: "3.5.0@sha256:c38c7f741e46ba5da3914eec9113b615bab779586ba4767605f0a3ebd494bcd2"
+  scyllaDBManagerAgentVersion: "3.5.0@sha256:12755969f1b3258572c520aa82c2f18db52287b3dc7055a977072284f234da16"
+  bashToolsImage: "registry.access.redhat.com/ubi9/ubi:9.5-1745854298@sha256:f4ebd46d3ba96feb016d798009e1cc2404c3a4ebdac8b2479a2ac053e59f41b4"
+  grafanaImage: "docker.io/grafana/grafana:11.6.0@sha256:62d2b9d20a19714ebfe48d1bb405086081bc602aa053e28cf6d73c7537640dfb" # Tracks scylla-monitoring/versions.sh GRAFANA_VERSION
+  grafanaDefaultPlatformDashboard: "scylladb-2025.1/scylla-overview.2025.1.json"
+  prometheusVersion: "v2.54.1" # Tracks scylla-monitoring/versions.sh PROMETHEUS_VERSION
```

Comment:
Scylla Monitoring 4.9.4 is using 3.1.x: https://github.com/scylladb/scylla-monitoring/blob/4.9.4/versions.sh#L104